features/download_item_list.feature

```diff
@@ -47,3 +41,25 @@ Feature: Managing Item Lists
     And I should see "Download Selected"
     And I should see "Download only files that match a particular regular expression"
     And I should see "Download Matches"
+
+
+  Scenario: Downloading more than 200 documents results in an error
+    Given "researcher@intersect.org.au" has item lists
+      | name  |
+      | Test1 |
+    And the item list "Test1" has 101 remote documents
+    When I am on the item list page for "Test1"
+    And I click "Download as ZIP"
+    And I press "Download All"
+    Then I should see "Zip download is limited to 200 files"
+
+  Scenario: Remote documents can be downloaded not result in an error
+    Given "researcher@intersect.org.au" has item lists
+      | name  |
+      | Test1 |
+    And the item list "Test1" has 101 remote documents
```

Comment:
I changed the step and didn't update the description. It used to create an item and each item had a single document, but I changed it so each item had two documents so I could use it with other steps (one with a .txt file and one without an extension). Updated the step to read:
'And the item list "Test1" has 101 items with two remote documents each'